### DISCUSSION TITLE
Could not send status to leader on Intel NUC

### DISCUSSION THREAD

**Discussion by CableMate** [nzymedefense/nzyme#1000] [Questions]
This is the successor of post "Could not send status to leader on Raspi #992".

I installed nzyme alpha 9 on an Intel NUC 10. The WLAN-Hardware is an AWUS036ACM.

Interesting is: I still get the tap log enteries "Could not send status to leader".
The NUC has 8GB available. 3.7GB are used and the 15 minute load is 2.34. 

I think, a system overload cannot be the reason for the timeout.

The issue seems not to have an effect visible in the GUI.

**Comment by mavacpjm** [comment]
I am getting the same on my Ubuntu 22.04 Leader from a RPI3 TAP on same network. Any idea how to trouble shoot? To add I am receiving an - **error trying to connect: error 14**

**SOLVED** Changed nzyme-tap.conf to allow for accept_insecure_certs = TRUE

Now it works perfect!

CLOSED

**Comment by lennartkoopmann** [comment]
I would need the logs for that one as well.

**Reply by mavacpjm** [reply]
I fixed the issue by allowing insecure certs for now, ill grab the logs and push to you. I am running the LEADER internal only with no outside Internet connection.
